docs/runbooks/HPPNotReady.md

```diff
@@ -1,32 +1,48 @@
+<!-- Edited by Jiří Herrmann, 3 Nov 2022 -->
+
 # HPPNotReady
 
 ## Meaning
 
-The hostpath-provisioner (HPP) is used to dynamically provision hostPath volumes to provide storage for PVCs.  
-The HPPNotReady alert is indicative of a HPP installation being in a degraded state;  
+The `hostpath-provisioner` (HPP) is used to dynamically provision hostPath volumes to provide storage for persistent volume claims (PVCs).  
```

Comment:
"The `hostpath-provisioner` (HPP) is used to dynamically provision..." > "The hostpath provisioner (HPP) dynamically provisions ...". 

"hostPath volumes" > "hostpath volumes". See [Configuring local storage](https://docs.openshift.com/container-platform/4.11/virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.html).